bot/cogs/reminders.py

```diff
@@ -99,10 +99,13 @@ def __init__(self, bot: Bot):
         embed.colour = Colour.blurple()
         embed.set_author(
             icon_url=Icons.remind_blurple,
-            name="It has arrived!"
-        )
+            name="It has arrived!")
 
-        embed.description = f"Here's your reminder: `{reminder['content']}`"
+        if "jump_url" in reminder:  # keep backward compatibility
+            embed.description = (f"Here's your reminder: `{reminder['content']}`.\n"
+                                 f"Jump back when you created the reminder : {reminder['jump_url']}")
```

Comment:
```suggestion
                                 f"Jump back to when you created the reminder : {reminder['jump_url']}")
```

Reply:
Why don't you turn the text into a clickable URL instead of creating a longer sentence by appending the message URL as text? The embed content allows for the use of `[text](url)` markdown and I think it would look much cleaner that way.